### ISSUE TITLE
latest update broke taskbar blur

### ISSUE THREAD

**Issue by watermelonbandit** [safak45xx/Blurfox#27]
Updated (well firefox did) my firefox a couple days ago and now there is no blur where the tabs are. Blur appears to work fine in the menus though

<img width="718" alt="Screenshot 2024-05-19 at 12 37 42 AM" src="https://github.com/safak45xx/Blurfox/assets/7614846/3d7a6edd-4baa-40b7-8b74-2f67785386c8">

**Comment by safak45xx** [comment]
Yes, I am aware of the issue. Firefox version 126 is causing problems in the code.

**Comment by safak45xx** [comment]
I haven't been able to identify the cause of the issue yet, but you can avoid the problem by using version 125.
https://ftp.mozilla.org/pub/firefox/releases/125.0.3/

**Comment by kratoz29** [comment]
> I haven't been able to identify the cause of the issue yet, but you can avoid the problem by using version 125. https://ftp.mozilla.org/pub/firefox/releases/125.0.3/

Huh, I tried to downgrade but it did not open, thankfully it booted when I downloaded 126 again 😅

Any ideas?
